### PR TITLE
Add documentation on how to apply/mark migrations to existing project…

### PR DESCRIPTION
**Pull request by tsdogs**
…s #172

This commit adds some documentation on how to migrate an existing database from dektrium\yii2-user to this project.

I've used this commands to migrate some of my projects and seem to complete the documentation.

### CODE REVIEW

**Comment by tonydspaniard** [comment]
@maxxer This looks ok to me to be merged.

**Comment by maxxer** [comment]
there are few remarks I wanted to be fixed before merging. but we can do it ourselves

**Comment by tonydspaniard** [comment]
@tsdogs our great cooperator @maxxer is on holidays... can you review again if there is anything missing? I can't actually point it. Maybe @maxxer can guide me and I'll do it?

**Comment by tsdogs** [comment]
Just got back from Holliday myself, will check laterali andò reporter.

**Comment by tsdogs** [comment]
@tonydspaniard I have checked but @maxxer did not mention what the "few remarks" were so I cannot check for him.
On my side everything seems to be working as described in the documentation. I have migarted successfully many projects without issues.

**Comment by maxxer** [comment]
I added the notes to the PR

YetOpen S.r.l. - https://www.yetopen.it/
Via Salerno 18 - 23900 Lecco - ITALY -
Tel +39 0341 220 205 - Fax +39 178 6070 222

Think green - Non stampare questa e-mail se non necessario / Don't print this email unless necessary

-------- D.Lgs. 196/2003 e GDPR 679/2016 --------
Tutte le informazioni contenute in questo messaggio sono riservate ed a uso esclusivo del destinatario.
Tutte le informazioni ivi contenute, compresi eventuali allegati, sono da ritenere confidenziali e riservate secondo i termini
del vigente D.Lgs. 196/2003 in materia di privacy e del Regolamento europeo 679/2016 - GDPR - e quindi ne e' proibita l'utilizzazione ulteriore non autorizzata.
Nel caso in cui questo messaggio Le fosse pervenuto per errore, La invitiamo ad eliminarlo senza copiarlo, stamparlo, a non inoltrarlo a terzi e ad avvertirci non appena possibile.
Grazie.

Confidentiality notice: this email message including any attachment is for the sole use of the intended recipient and may contain confidential and privileged information;
pursuant to Legislative Decree 196/2003 and the European General Data Protection Regulation 679/2016 - GDPR - any unauthorized review, use, disclosure or distribution
is prohibited. If you are not the intended recepient please delete this message without copying, printing or forwarding it to others, and alert us as soon as possible.
Thank you.

**Comment by maxxer** [comment]
I think the only thing missing is the `composer add` which I don't think exists

File: docs/installation/migration-guide-from-dektrium-tools.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ composer remove dektrium/yii2-user
 composer remove dektrium/yii2-rbac
 ```
 
+## Install yii2-usuario
+```
+composer add 2amigos/yii2-usuario
```
Comment (tonydspaniard) [inline docs/installation/migration-guide-from-dektrium-tools.md]: Shouldn't be `composer require 2amigos/yii2-usuario`?

Comment (tsdogs) [reply docs/installation/migration-guide-from-dektrium-tools.md]: Yes it should be require.

**Comment by tonydspaniard** [comment]
Thank you @tsdogs

**Pull request merged**